### PR TITLE
Renamed 6.0.0 to 6.x

### PR DESCRIPTION
**Pull request by stevengill**

### CODE REVIEW

**Comment by stevengill** [comment]
Still need to finish incorporating feedback from #484

File: www/_data/toc/zh-6-x-manual.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- {name: "\u5916\u639B\u7A0B\u5F0F\u958B\u767C\u6307\u5357", url: !!python/unicode '/docs/zh/6.x/guide/hybrid/plugins/index.html'}
+- {name: "\u5916\u639B\u7A0B\u5F0F\u898F\u7BC4", url: !!python/unicode '/docs/zh/6.x/plugin_ref/spec.html'}
+- {name: "\u96B1\u79C1\u6307\u5357", url: !!python/unicode '/docs/zh/6.x/guide/appdev/privacy/index.html'}
+- {name: "\u5B89\u5168\u6307\u5357", url: !!python/unicode '/docs/zh/6.x/guide/appdev/security/index.html'}
+- {name: "\u5E73\u81FA\u548C\u5916\u639B\u7A0B\u5F0F\u7248\u672C\u7BA1\u7406", url: !!python/unicode '/docs/zh/6.x/platform_plugin_versioning_ref/index.html'}
+- {name: "\u767D\u540D\u55AE\u6307\u5357", url: !!python/unicode '/docs/zh/6.x/guide/appdev/whitelist/index.html'}
+- {name: "\u5B58\u5132", url: !!python/unicode '/docs/zh/6.x/cordova/storage/storage.html'}
+- {name: "\u4E8B\u4EF6", url: !!python/unicode '/docs/zh/6.x/cordova/events/events.html'}
+- {name: "\u5916\u639B\u7A0B\u5F0F\u7684 Api", url: !!python/unicode '/docs/zh/6.x/cordova/plugins/pluginapis.html'}
+- {name: "\u95DC\u9375\u5B57\u7D22\u5F15", url: !!python/unicode '/docs/zh/6.x/page_index.html'}
```
Comment (dblotsky) [inline www/_data/toc/zh-6-x-manual.yml]: This change should only be a rename: there are no `/docs/XX/YYY/` prefixes in the ToC files anymore.

File: tools/bin/incrementversion.js [hunk 2]
```diff
@@ -36,9 +36,11 @@ function isValidVersion(version) {
     if (version == DEV_VERSION_NAME) {
         return false;
     }
+    /*
     if (!semver.valid(version)) {
```
Comment (dblotsky) [inline tools/bin/incrementversion.js]: Please add a `validVersion` function here, just with some simpler logic. A test `\d\.x` should probably be fine.

File: tools/bin/incrementversion.js [hunk 3]
```diff
@@ -71,10 +73,10 @@ function main () {
     }
 
     // only create greater versions
-    if (!semver.gt(newVersion, oldVersion)) {
+    /*if (!semver.gt(newVersion, oldVersion)) {
```
Comment (dblotsky) [inline tools/bin/incrementversion.js]: Please make this a simple function: something like just a check for greater major version number.

File: www/.htaccess [hunk 4]
```diff
@@ -41,6 +41,9 @@ RewriteRule ^.*docs/(\w\w)/latest$ {{site.baseurl}}/docs/$1/latest/ [R=302,L]
 #       resource, but the browser won't change the URI (no redirects followed).
 RewriteRule ^.*docs/(\w\w)/latest/(.*)$ {{site.baseurl}}/docs/$1/{{site.latest_docs_version}}/$2 [L]
 
+# rewrite 6.0.0 links to point to 6.x links
+RewriteRule ^.*docs/(\w\w)/6.0.0/(.*)$ {{site.baseurl}}/docs/$1/6.x/$2 [L]
```
Comment (dblotsky) [inline www/.htaccess]: Please make this a 301 redirect.

**Comment by dblotsky** [comment]
@stevengill you can close this PR. I've done the work in #520 and I'll merge it tomorrow.

**Comment by dblotsky** [comment]
@stevengill please close this PR when you have a moment. :) The changes have been applied.

**Pull request closed without merging**